### PR TITLE
Queue with stacks

### PR DESCRIPTION
**Pull request by herndont**
- [x] Created the pseudo Queue and added the objects
- [x] Wrote the code in the queues with stacks class
- [x] Tested the code for happy path and expected failure

### CODE REVIEW

**Comment by herndont** [comment]
I tried to remove the idea files before this pull request but it said there was nothing that matched. If this didn't work please let me know so that I can go over this with you and make sure that I am doing this correctly. Thanks

File: src/main/java/QueueWithStacks/PseudoQueue.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package QueueWithStacks;
+
```
Comment (JessLovell) [inline src/main/java/QueueWithStacks/PseudoQueue.java:2]: 🤷‍♀

File: src/main/java/stacksandqueues/Stack.java [hunk 2]
```diff
@@ -27,4 +27,12 @@ public int peek() {
         }
         return this.top.data;
     }
+
+    public boolean isEmpty() {
```
Comment (JessLovell) [inline src/main/java/stacksandqueues/Stack.java:31]: Did you add tests for this method?

File: src/test/java/QueueWithStacksTest.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import QueueWithStacks.QueueWithStacks;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class QueueWithStacksTest {
+
+    @Test
+    public void enqueueTest() throws Exception {
+//  Happy Path Test
+        QueueWithStacks testQ = new QueueWithStacks();
```
Comment (JessLovell) [inline src/test/java/QueueWithStacksTest.java:10]: This enqueue test should test each time you enqueue.

**Pull request merged**